### ISSUE TITLE
Fatal Error - Maximum execution time of 5 seconds exceeded

### ISSUE THREAD

**Issue by opiums9** [Sarjuuk/aowow#275]
After running the application in the CLI, I get this error after a while
```
"D:\xampp\php\php.exe" aowow --firstrun
...
11:56:16  [OK]    created file datasets/frfr/p-recipes-197
11:56:16  [OK]    created file datasets/dede/p-recipes-197
11:56:16  [OK]    created file datasets/zhcn/p-recipes-197
11:56:16  [OK]    created file datasets/eses/p-recipes-197
11:56:16  [OK]    created file datasets/ruru/p-recipes-197
11:56:16  [OK]    created file datasets/enus/p-recipes-sec
11:56:16  [OK]    created file datasets/frfr/p-recipes-sec
11:56:16  [OK]    created file datasets/dede/p-recipes-sec
11:56:16  [OK]    created file datasets/zhcn/p-recipes-sec
11:56:16  [OK]    created file datasets/eses/p-recipes-sec
11:56:16  [OK]    created file datasets/ruru/p-recipes-sec
11:56:16  [OK]    created file datasets/enus/achievements
11:56:16  [OK]    created file datasets/frfr/achievements
11:56:16  [OK]    created file datasets/dede/achievements
11:56:16  [OK]    created file datasets/zhcn/achievements
11:56:16  [OK]    created file datasets/eses/achievements
11:56:16  [OK]    created file datasets/ruru/achievements
11:56:16          applying 89 baseline exclusions
Fatal Error - Maximum execution time of 5 seconds exceeded @ D:\xampp\htdocs\includes\libs\DbSimple\Mysqli.php:163
```
What could it be?

**Comment by Jinnaix** [comment]
This has nothing to do with AoWoW itself but with your webserver / php settings you currently run on.
Seems like the default time a script is allowed to run on your current setup is 5 seconds - way to low to setup aowow.

**Comment by Sarjuuk** [comment]
i think i got it. 
5sec is set from the previous step (writing achievement data to file) and the next step (compiling the exclusions file) doesn't reset it because it usually takes mere ms.
It should probably get it's own limit and if it's still exceeded .. well than there is another issue.

**Comment by Sarjuuk** [comment]
maybe fixed in 7a74c3644847b9fb25b9083656c4df7e62dfcf96 ?

In any case if it still needs more time we should debug this step further.

**Comment by qyh214** [comment]
now is Fatal Error - Maximum execution time of 2 seconds exceeded @ xxxx\includes\libs\DbSimple\Mysqli.php:162

commit：  56e70e22bba81692670199c292c231ab0c113cdb
